test/recipes/70-test_quic_multistream.t

```diff
@@ -34,7 +34,7 @@ ok(run(test(["quic_multistream_test",
 
 SKIP: {
     skip "no qlog", 1 if disabled('qlog');
-    skip "not running CI tests", 1 if $ENV{OSSL_RUN_CI_TESTS} != "1";
+    skip "not running CI tests", 1 if ($ENV{OSSL_RUN_CI_TESTS} // "") ne "1";
```

Comment:
Wouldn't it be easier like this?

``` perl
    skip "not running CI tests", 1 unless $ENV{OSSL_RUN_CI_TESTS};
```

(if the environment is undefined, the value will be `undef`, which is false)